plugins/foreman_remote_execution/0.1/index.md

```diff
@@ -0,0 +1,407 @@
+above, the value can be specified by user when they trigger the execution.
+
+A few examples of tasks that this plugin helps you achieve:
+
+- Install/update/remove a package
+- Bootstrap a configuration management agent
+- Trigger a Puppet/Salt/Chef run
+
+## 1.1 Components
+
+The whole remote execution system is pluggable, making it easy to add more
+providers of communication later. Therefore it's split into several components
+that must be installed on Foreman and the Smart Proxy.
+
+<table class="table table-bordered table-condensed">
+  <tr>
+    <th>Plugin</th>
+    <th>Description</th>
+  </tr>
+  <tr>
+    <td>foreman_remote_execution</td>
+    <td>Required, contains the UI part and currently also SSH provider, depends on foreman-tasks</td>
+  </tr>
+  <tr>
+    <td>foreman_tasks</td>
+    <td>Required, keep status of all executed commands, allows debugging if something goes wrong</td>
+  </tr>
+  <tr>
+    <td>smart_proxy_remote_execution_ssh</td>
+    <td>Smart Proxy plugin, required for using SSH provider, proxies the communication between Foreman and hosts, requires smart_proxy_dynflow</td>
+  </tr>
+  <tr>
+    <td>smart_proxy_dynflow</td>
+    <td>Smart Proxy plugin, runs the jobs asynchronously and keeps status of commands executed via this proxy. This plugins uses relational SQL DB (sqlite).</td>
+  </tr>
+</table>
+
+
+## 1.2 Compatability
+
+Each component is packaged for platforms that Foreman is officially packaged
+for. Since both Foreman and Smart Proxy plugins requires Dynflow, Ruby 1.9+ is
+required.
+
+### 1.2.1 foreman_remote_execution
+
+<table class="table table-bordered table-condensed">
+  <tr>
+    <th>Foreman</th>
+    <th>foreman_remote_execution</th>
+  </tr>
+  <tr>
+    <td>1.9.x</td>
+    <td>0.0.x</td>
+  </tr>
+  <tr>
+    <td>1.10.x</td>
+    <td>0.1.x</td>
+  </tr>
+</table>
+
+### 1.2.2 smart_proxy_remote_execution_ssh
+
+<table class="table table-bordered table-condensed">
+  <tr>
+    <th>Smart Proxy</th>
+    <th>smart_proxy_remote_execution_ssh</th>
+    <th>Notes</th>
+  </tr>
+  <tr>
+    <td>1.9.x</td>
+    <td>0.0.x</td>
+    <td>No compatability with EL6 or Ubuntu 12.04</td>
+  </tr>
+  <tr>
+    <td>1.10.x</td>
+    <td>0.0.x</td>
+    <td>No compatability with EL6 or Ubuntu 12.04</td>
+  </tr>
+</table>
+
+# 2. Installation
+
+## 2.1 Plugin Installation
+
+This chapter covers a clean Foreman and plugin installation. We assume the
+smart proxy will be installed on the same host, however it's not a requirement.
+Please adapt according to your needs.
+
+Start by installing the Foreman nightly or 1.10 repository and EPEL7, see
+[Quickstart instructions](/manuals/latest/index.html#2.Quickstart) and [Foreman
+manual](/manuals/latest/index.html#3.InstallingForeman) for more information.
+
+### With Foreman Installer
+
+You can install both the Foreman and Smart Proxy plugin with the installer:
+
+    foreman-installer --enable-foreman-plugin-remote-execution\
```

Comment:
Hmm, good catch, updated, and fixed over here for next release https://github.com/theforeman/puppet-foreman/pull/375